quiz.py

```diff
@@ -1,6 +1,56 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Chiara A
+Date: 2024/02/01
+Description: This program is a multiple quiz game!
+"""
+
+print("Multiple choice quiz game!\n")
+
+
+user_score = 0
+total_questions = 4
+questions_correct = 0
+
+question_one = input("What is my nick name?: \na) Chiara \nb) kiki \nc) chichi \nd) ki \n")
+
+if question_one == "b" or question_one == "B":
+    print("Correct!\n")
+    new_score = user_score + 1
+    questions_correct = questions_correct + 1
```

Comment:
There are two score counters. It looks like only `questions_correct` is used in the final calculation. What is `new_score` and `user_score` used for? if they are superfluous, you should remove them.